packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -188,6 +189,9 @@ export const useArbTokenBridge = (
     }
 
     const ethBridger = await EthBridger.fromProvider(l2.provider)
+    const parentChainBlockTimestamp = Math.floor(
```

Comment:
Why is `Math.floor` needed here since it will never be a decimal

Reply:
hmm I was getting a decimal value for some reason before but might have been a mistake on my part, removing math floor